test/events_test.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require_relative "helper"
+
+describe Kasket::Events do
+  before do
+    @previous = Kasket::CONFIGURATION[:events_callback]
+    Kasket::Events.remove_instance_variable(:@fn) if Kasket::Events.instance_variable_defined?(:@fn)
+  end
+
+  after { Kasket::CONFIGURATION[:events_callback] = @previous }
+
+  describe "::report" do
```

Comment:
We usually use `.` when describing class methods (and `#` for instance methods):
```suggestion
  describe ".report" do
```